src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -99,4 +132,15 @@ Enable the DNS challenge for a domain managed on Cloudflare with account credent
 tls {
 	dns cloudflare {env.CLOUDFLARE_API_TOKEN}
 }
-```
\ No newline at end of file
+```
+
+Enable TLS Client Authentication
```

Comment:
To mirror the only remaining comment from @mholt on the other PR https://github.com/caddyserver/website/pull/49#discussion_r436088373

> Explain what kind of authentication, or what is necessary and sufficient to auth

This example could use some additional explanation

Reply:
Does the updated example explanation work? :)

Reply:
Yep! Looks good to me :smile: